### DISCUSSION TITLE
QSM creating support

### DISCUSSION THREAD

**Discussion by j-miszczyszyn** [lmterryn/ITSMe#5] [Ideas]
Hi,
Have you already considered the possibility of creating a function that creates a QSM model inside the package ?

I think your package is very good and I used it during my master's thesis in which I analyzed stand structure based on LiDAR HLS Geoslam Zeb Horizon. However, I couldn't create a QSM model because I don't know MATLAB or have its license. 

If not. then can you recommend a package that creates a QSM model not based on MATLAB language, but on R? I found [https://github.com/umr-amap/aRchi.git](url) but still testing this solution.

Greetings,
Jakub

**Comment by lmterryn** [comment]
Hi Jakub,

Sorry for the (very) late reply. At the moment there are no plans to add functions to create QSMs. Only to import them and extract structural metrics from them. Maybe you could try another open-source QSM algorithm like Ray Cloud Tools (https://github.com/csiro-robotics/raycloudtools). We saw it produces similar results (total volume) as the initial version of treeQSM (v2.0) and when compared to destructive harvest data. 

Cheers,
Louise
